### PR TITLE
Meeting summaries for 2016-07-21

### PR DESCRIPTION
**Pull request by G1lius**

### CODE REVIEW

File: _posts/en/meetings/2016-07-21-meeting.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+ 
+The Bitcoin Core team is working towards the 0.13.0 release ([full schedule](https://github.com/bitcoin/bitcoin/issues/7679)) and RC1 is [available](https://bitcoin.org/bin/bitcoin-core-0.13.0/test.rc1/) since 2016-07-20.
+
+### meeting comments
+ 
+RC1 got some [feedback](https://github.com/bitcoin/bitcoin/issues/8383) noticing that when encrypting the wallet, it uses the same HD seed, which means the HD seed has been unencrypted on the disk when the wallet was created. Jonasschnelli is working on a [fix][#8389] to create a new HD seed after encrypting the wallet.
+
+A common complaint is the lack of a way to export the HD seed. Jonasschnelli has a [pull request][#8206] which is easy to review and has a low impact that exports the seed to dumpwallet. Importing is a different issue with much more impact as the wallet currently doesn't support multiple seeds. This is a feature for 0.14.
+
+Luke-jr notes the default policy isn't performing as well as inadvisable policies for which he made a [pull request][#8388]. This is a pretty large change and requires some more discussion.
```
Comment (harding) [inline _posts/en/meetings/2016-07-21-meeting.md]: > the default policy isn't performing as well as inadvisable policies

I'm having difficulty parsing this clause; are there maybe some missing words or extra details you can add?

File: _posts/en/meetings/2016-07-21-meeting.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+| jeremyrubin   | [Jeremy Rubin][]          |
+| CodeShark     | [Eric Lombrozo][]         |
+| NicolasDorier | [Nicolas Dorier][]        | 
+| BlueMatt      | [Matt Corallo][]          |
+
+## Disclaimer
+ 
+This summary was compiled without input from any of the participants in the discussion, so any errors are the fault of the summary author and not the discussion participants.
+ 
+[#8389]: https://github.com/bitcoin/bitcoin/pull/8389-
```
Comment (harding) [inline _posts/en/meetings/2016-07-21-meeting.md]: Should all of the links here have a dash after them?

File: _posts/en/meetings/2016-07-21-meeting.md (hunk 1 above)
Comment (G1lius) [inline _posts/en/meetings/2016-07-21-meeting.md]: It's a literal quote. but I'll put in more details.

**Comment by harding** [comment]
@G1lius thanks!

**Comment by btcdrak** [comment]
LGTM

@harding I pinged you on Slack :)

File: _posts/en/meetings/2016-07-21-meeting.md [hunk 3]
```diff
@@ -38,7 +38,7 @@ RC1 got some [feedback](https://github.com/bitcoin/bitcoin/issues/8383) noticing
 
 A common complaint is the lack of a way to export the HD seed. Jonasschnelli has a [pull request][#8206] which is easy to review and has a low impact that exports the seed to dumpwallet. Importing is a different issue with much more impact as the wallet currently doesn't support multiple seeds. This is a feature for 0.14.
 
-Luke-jr notes the default policy isn't performing as well as inadvisable policies for which he made a [pull request][#8388]. This is a pretty large change and requires some more discussion.
+Luke-jr notes the new default policy of using blockmaxweight isn't performing as well as using blockmaxsize in the current environment. He made a [pull request][#8388] to change this. This is a pretty large change and requires some more discussion.
```
Comment (luke-jr) [inline _posts/en/meetings/2016-07-21-meeting.md]: This is wrong. The default policy isn't changed (and shouldn't). It's blockmaxweight that is inadvisable and currently performing better.

Comment (G1lius) [inline _posts/en/meetings/2016-07-21-meeting.md]: Release notes read:  
-For optimal runtime performance, miners using this release should specify
 -`-blockmaxweight` on the command line, and not specify `-blockmaxsize`.
 -Additionally (or only) specifying `-blockmaxsize`, or relying on default
 -settings for both, may result in performance degradation, as the logic to
 -support `-blockmaxsize` performs additional computation to ensure that
 -constraint is met.

I read that as it being recommended to use blockmaxweight.

Comment (luke-jr) [inline _posts/en/meetings/2016-07-21-meeting.md]: That's a problem with the current release notes. Miners should not be using blockmaxweight at this time (nor when 0.13.0 is released).

**Pull request merged**